PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -82,22 +82,44 @@
                             "2*filter('*MediumChargedIso*') + " \
                             "4*filter('*TightChargedIso*') + " \
-                            "8*filter('*TightOOSCPhotons*') + " \
-                            "16*filter('*Hps*') + " \
-                            "32*filter('hltSelectedPFTau*MediumChargedIsolationL1HLTMatched*') + " \
-                            "64*filter('hltDoublePFTau*TrackPt1*ChargedIsolation*Dz02Reg') + " \
-                            "128*filter('hltOverlapFilterIsoEle*PFTau*') + " \
-                            "256*filter('hltOverlapFilterIsoMu*PFTau*') + " \
-                            "512*filter('hltDoublePFTau*TrackPt1*ChargedIsolation*')"),
-            qualityBitsDoc = cms.string("1 = LooseChargedIso, 2 = MediumChargedIso, 4 = TightChargedIso, 8 = TightID OOSC photons, 16 = HPS, 32 = single-tau + tau+MET, 64 = di-tau, 128 = e-tau, 256 = mu-tau, 512 = VBF+di-tau"),            
+                            "8*filter('*DeepTau*') + " \
+                            "16*filter('*TightOOSCPhotons*') + " \
+                            "32*filter('*Hps*') + " \
+                            "64*filter('hlt*DoublePFTau*TrackPt1*ChargedIsolation*Dz02*') + " \
+                            "128*filter('hlt*DoublePFTau*DeepTau*L1HLTMatched') + " \
+                            "256*filter('hlt*OverlapFilterIsoEle*WPTightGsf*PFTau') + " \
+                            "512*filter('hlt*OverlapFilterIsoMu*PFTau*') + " \
+                            "1024*filter('hlt*SelectedPFTau*L1HLTMatched') + " \
+                            "2048*filter('hlt*DoublePFTau*TrackPt1*ChargedIso*') + " \
+                            "4096*filter('hlt*DoublePFTau*Track*ChargedIso*AgainstMuon') + " \
+                            "8192*filter('hltHpsSinglePFTau*HLTMatched') + " \
+                            "16384*filter('hltHpsOverlapFilterDeepTauDoublePFTau*PFJet*') + " \
+                            "32768*filter('hlt*Double*ChargedIsoDisplPFTau*Dxy*') + " \
+                            "65536*filter('*Monitoring') + " \
+                            "131072*filter('*Reg') + " \
+                            "262144*filter('*L1Seeded') + " \
+                            "524288*filter('*1Prong')"),
+            qualityBitsDoc = cms.string("1 = LooseChargedIso, 2 = MediumChargedIso, 4 = TightChargedIso, 8 = DeepTau, 16 = TightID OOSC photons, 32 = HPS, 64 = charged iso di-tau, 128 = deeptau di-tau, 256 = e-tau, 512 = mu-tau, 1024 = single-tau/tau+MET, 2048 = run 2 VBF+ditau, 4096 = run 3 VBF+ditau, 8192 = run 3 double PF jets + ditau, 16384 = di-tau + PFJet, 32768 = Displaced Tau, 65536 = Monitoring, 131072 = regional paths, 262144 = L1 seeded paths, 524288 = 1 prong tau paths"),            
+        ),
+        cms.PSet(
+            name = cms.string("BoostedTau"),
+            id = cms.int32(6),
+            sel = cms.string("type(85) && pt > 120 && coll('hltAK8PFJetsCorrected')"), 
```

Comment:
If I understand correctly with `hltAK8PFJetsCorrected` selection this collection will contain all AK8 PF jets, which is too much. Can we use the same filter as below to pre-select? In the future, it will grow as new paths will arrive.

Reply:
Which one do you mean with below? The jet one seems looser to me than this one.

Reply:
Ah, now I see. That's fair enough, I'll make the change